### PR TITLE
Update calibration examples

### PR DESCRIPTION
**Pull request by mbaudin47**
* Moved all calibration examples (including full bayesian examples) into the "Calibration" section,
* Fixed the content of the calibration examples,
* Added references from the calibration theory to the corresponding examples.

### CODE REVIEW

**Comment by mbaudin47** [comment]
There is a figure which should appear in the example:

https://github.com/openturns/openturns/blob/b968dc2ba13dda2d87aad23e2b2385ee626828cf/python/doc/examples/calibration/calibration_deflection_tube.ipynb#L18

but it does not appear in the output HTML rendering: do you understand why ?

**Comment by jschueller** [comment]
The rendering on github may differ from the sphinx rendering.

**Comment by mbaudin47** [comment]
There is an unrelated bug on appveyor : "C:\projects\openturns\lib\src\Base\Stat\FieldImplementation.cxx(425): error C2668: 'std::sqrt' : ambiguous call to overloaded function [C:\projects\openturns\build\lib\src\OT.vcxproj]"

**Comment by mbaudin47** [comment]
This is not ready for a review: the numerical results are different.

**Comment by mbaudin47** [comment]
The appveyor failure is caused by the FieldImplementation error. This is why I think that this PR is good to go.

File: python/doc/examples/calibration/calibration_chaboche.ipynb [hunk 1]
```diff
@@ -74,7 +60,16 @@
     "\\sigma_i = G(\\epsilon_i,R,C,\\gamma) + (\\epsilon_\\sigma)_i, \n",
     "$$\n",
     "for $i = 1,..., n$.\n",
-    "The observations are the couples $\\{(\\epsilon_i,\\sigma_i)\\}_{i=1,...,n}$, i.e. each observation is a couple made of the strain and the corresponding stress."
+    "The observations are the couples $\\{(\\epsilon_i,\\sigma_i)\\}_{i=1,...,n}$, i.e. each observation is a couple made of the strain and the corresponding stress.\n",
```
Comment (sofianehaddad) [inline python/doc/examples/calibration/calibration_chaboche.ipynb]: couples --> pairs ?

File: python/doc/examples/calibration/calibration_chaboche.ipynb [hunk 2]
```diff
@@ -74,7 +60,16 @@
...
+    "\n",
+    "## Thanks to\n",
+    "\n",
+    "- Antoine Dumas, Phiméca\n",
```
Comment (sofianehaddad) [inline python/doc/examples/calibration/calibration_chaboche.ipynb]: Please drop written accents

Comment (mbaudin47) [reply python/doc/examples/calibration/calibration_chaboche.ipynb]: Fixed.

**Comment by sofianehaddad** [comment]
Seems you still got suspicious commits
Please remove them using interactive rebase

 ``` 
   git fetch upstream
   git rebase -i upstream/master updateCalibrationExamples
```
and drop "merge" commits (b30dfda)

**Comment by mbaudin47** [comment]
> Seems you still got suspicious commits
> Please remove them using interactive rebase
> 
> ```
>   git fetch upstream
>   git rebase -i upstream/master updateCalibrationExamples
> ```
> 
> and drop "merge" commits ([b30dfda](https://github.com/openturns/openturns/commit/b30dfda0de02899c01d1bcf750dde5c5a0ae9f85))

Thanks for the advice ; here's what I did:
```
git fetch upstream
git rebase -i upstream/master updateCalibrationExamples
git rebase --continue
gedit python/doc/theory/data_analysis/bayesian_calibration.rst # Fixed a conflict
git add python/doc/theory/data_analysis/bayesian_calibration.rst
git rebase --continue
git push -f origin updateCalibrationExamples
```
This should be good... i hope !

**Comment by sofianehaddad** [comment]
@mbaudin47 looks good. Now we can focus on the PR ;-)

**Pull request merged**